src/data/git-quiz.ts

```diff
@@ -430,6 +430,16 @@ const gitQuiz = [
       "The git push -f command will force a push request. This is usually fine for pull request branches because nobody else should have cloned them.",
     Link: "https://www.freecodecamp.org/news/git-cheat-sheet/",
   },
+  {
+    Question: "Why is it impossible to make changes without Git knowing about it?"
```

Comment:
@deedeecee 
Add `,` comma at the end of line to resolve the linting issues.

Reply:
Thanks for the comment, sir. 
I'll make the necessary changes as you asked.

Reply:
@deedeecee Still the comma has not been added at the end of line no 434. Please read PR comments properly and make the changes.
I believe your PR will be approved, only If the PR does not have any linting issues. So fix the error in line no 434.